tests/test_project.py

```diff
@@ -565,6 +663,7 @@ def test_index(self):
         assert len(docs) == 2 * len(statepoints)
         assert len(set((doc['_id'] for doc in docs))) == len(docs)
 
+    @pytest.mark.xfail()
```

Comment:
I think it would be good to add a small comment for all the expected failures describing the reason. Those represent action items.